anesthetic/weighted_pandas.py

```diff
@@ -298,9 +315,16 @@ def compress(self, nsamples=None):
         data = np.repeat(self.to_numpy(), i, axis=0)
         index = self.index.repeat(i)
         df = DataFrame(data=data, index=index, columns=self.columns)
-        df.index = df.index.get_level_values('#')
+        df.index = df.index.droplevel('weights')
         return df
 
+    def _reduce(self, op, name, **kwargs):
+        answer = super()._reduce(op, name, **kwargs)
+        if kwargs['axis'] == 0:
+            answer = Series(answer).droplevel('weights')
+            answer.name = None
+        return answer
```

Comment:
Why is this needed?

Why is this not needed for `WeightedSeries`?

Reply:
Removed in latest update